### ISSUE TITLE
Support DynamoDB

### ISSUE THREAD

**Issue by hackerwins** [yorkie-team/yorkie#533]
**Description**:

Support DynamoDB.

Yorkie cluster stores data, including documents and changes, in DB. Currently, it supports MongoDB and in-memory DB for testing purposes. https://github.com/yorkie-team/yorkie/tree/main/server/backend/database

**Why**:

Using managed DB like DynamoDB on AWS is an effective way to operate the cluster in the cloud, compared to self-hosting. Supporting DynamoDB would ensure the stable operation of the demo cluster, which is currently hosted on AWS.

**Comment by sullyvannunes** [comment]
I am interested in this issue. Can I work on it?

**Comment by hackerwins** [comment]
@sullyvannunes Of course. If you have any questions, please feel free to let us know.

**Comment by hackerwins** [comment]
Introducing a DynamoDB implementation is currently a low priority, so I'll close it for now.
